src/index.js

```diff
@@ -20,6 +22,13 @@ let promptIntent = {
 };
 let followLogs = false;
 
+let mainMenu ="";
+if (settings.enableDisableForging) {
```

Comment:
I see this could be improved for future scalability... Imagine tomorrow we have a secondary optional feature; this would probably create a complex if condition.
I would probably do it in this way:


in the index.js
```
const { cleanIntent, setIntent, sendChunkedMessage, createMenu } = require("./lib/utils");
let menu = createMenu();
```

and in lib/utils.js
```
const createMenu = () => {
  // Pseudo code
 // get the base menu
 // if enableDisableForging -> add to the base menu the enableDisableForging buttons
 // return the menu
};
```